sections/earn/StakeGrid.tsx

```diff
@@ -45,7 +45,14 @@ const StakeGrid = () => {
 		<GridContainer>
 			<GridData title="Your Yield / Day" value={yieldPerDay} hasKwentaLogo />
 			<GridData title="Your Rewards" value={truncateNumbers(earnedRewards, 4)} hasKwentaLogo>
-				<Button fullWidth variant="flat" size="sm" style={{ marginTop: 10 }} onClick={handleClaim}>
+				<Button
+					fullWidth
+					variant="flat"
+					size="sm"
+					style={{ marginTop: 10 }}
+					disabled={!toWei(earnedRewards).gt(0)}
```

Comment:
bug fixes

Reply:
This is a good improvement in UX. Thanks for the initiative. I just want to emphasize that it's not a bug because the staking rewards contract supports the `zero reward claim` scenario.

Reply:
I can not find the contract so that I don't know what the scenario is, claim zero reward means users can get nothing but cost additional gas fee?

Reply:
Found it:

https://optimistic.etherscan.io/address/0x6077987e8e06c062094c33177Eb12c4A65f90B65#code

in `StakeRewards.sol` file, `getReward` function.